kernel/cooking.ml

```diff
@@ -122,15 +122,10 @@ type internal_abstr_inst_info = Univ.Instance.t * int list * int
 type cooking_cache = {
   cache : internal_abstr_inst_info RefTable.t;
   info : cooking_info;
+  named_ctx : named_context Lazy.t;
   rel_ctx : rel_context Lazy.t;
```

Comment:
Are the lazys really useful? AFAICT we're going to force them immediately after every create_cache.